src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -5,20 +5,23 @@ import { ServiceHealth } from '../../types/Health';
 import { DisplayMode, HealthIndicator } from '../../components/Health/HealthIndicator';
 import MissingSidecar from '../../components/MissingSidecar/MissingSidecar';
 import { PromisesRegistry } from '../../utils/CancelablePromises';
+import { ObjectValidation } from '../../types/IstioObjects';
+import { ConfigIndicator } from '../../components/ConfigValidation/ConfigIndicator';
 
 interface Props {
   item: ServiceListItem;
 }
 interface State {
   health?: ServiceHealth;
+  validation: ObjectValidation;
 }
 
 export default class ItemDescription extends React.PureComponent<Props, State> {
   private promises = new PromisesRegistry();
 
   constructor(props: Props) {
     super(props);
-    this.state = { health: undefined };
+    this.state = { health: undefined, validation: {} as ObjectValidation };
```

Comment:
Perhaps I am not fully following this. I see that the validation object is defined in the state interface and there isn't any instruction writting any value different thatn {}. Who is the one adding the actual Validations here?